helpdesk_mgmt/demo/helpdesk_demo.xml

```diff
@@ -25,6 +25,15 @@
         <field eval="&quot;General Alias for tickets&quot;" name="alias_name" />
         <field ref="model_helpdesk_ticket" name="alias_model_id" />
     </record>
+    <record id="mail_alias_2" model="mail.alias">
```

Comment:
Why changing this?

Reply:
Thanks @pedrobaeza 
In the demo when you have the same alias for all the team, it's difficult to make test because when you change the alias_name in the view it actually change the alias name on *all* the team
That's why I added 2 alias to have one per team